tests/data/fixture_data.py

```diff
@@ -1,34 +1,40 @@
 from typing import Dict, List, Union
+from baracoda.formats import HeronCogUkIdFormatter, GenericBarcodeFormatter
 
-PREFIXES: List[Dict[str, Union[str, bool]]] = [
+PREFIXES: List[Dict[str, Union[str, object]]] = [
     {
         "prefix": "SANG",
         "sequence_name": "heron",
-        "convert": True,
+        "formatter_class": HeronCogUkIdFormatter,
```

Comment:
Liking the idea of formatters. When I initially did this we only had the one format so this is a natural progression and makes Baracode more flexible without adding complexity. Well done.